lib/collection/src/operations/snapshot_ops.rs

```diff
@@ -81,7 +82,17 @@ pub struct SnapshotRecover {
     pub api_key: Option<String>,
 }
 
+fn snapshot_description_example() -> SnapshotDescription {
+    SnapshotDescription {
+        name: "prefix-cache".to_string(),
```

Comment:
Names are usually in a format like this:

```suggestion
        name: "my-collection-3766212330831337-2024-07-22-08-31-55.snapshot".to_string(),
```